.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,29 @@
+name: goreleaser
+
+on:
+  pull_request:
+  push:
```

Comment:
We probably want to restrict this to only tagged releases. We don't need to be generating binaries for each push in a PR.